### PR TITLE
DO NOT MERGE - bff users migration command

### PR DESCRIPTION
**Pull request by pdeziel**
### Scope of changes

This bff command performs a migration from an Auth0 domain to a bff database, making sure that all users in Auth0 who have a current `orgid` have an organization record in the database. Additionally, it will make them a collaborator on the organization record so that they can login.

### Type of change

- [ ] bug fix
- [ ] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [ ]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [ ]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: cmd/bff/main.go [hunk 1]
```diff
@@ -98,6 +106,65 @@ func main() {
...
+						Aliases: []string{"C"},
+						Usage:   "specify the path to the certs to use when connecting to the database",
+						EnvVars: []string{"GDS_BFF_DATABASE_CERT_PATH"},
+					},
+					&cli.StringFlag{
+						Name:    "pool-path",
+						Aliases: []string{"p"},
+						Usage:   "specify the path to the certs pool to use when connecting to the database",
+						EnvVars: []string{"GDS_BFF_DATABASE_POOL_PATH"},
+					},
```
Comment (pdeziel) [inline cmd/bff/main.go:160]: If we run this in the bff pod it should pick up the configuration from the environment.

File: cmd/bff/main.go [hunk 2]
```diff
@@ -228,6 +295,97 @@ func announce(c *cli.Context) (err error) {
...
+
+		if appdata.OrgID != "" {
+			// Retrieve user's organization from the database
+			var orgID uuid.UUID
+			if orgID, err = models.ParseOrgID(appdata.OrgID); err != nil {
+				return cli.Exit(err, 1)
+			}
+			var org *models.Organization
+			if org, err = db.RetrieveOrganization(orgID); err != nil {
+				if errors.Is(err, storeerrors.ErrEntityNotFound) {
```
Comment (bbengfort) [inline cmd/bff/main.go:346]: I'm glad you're catching this - but out of curiosity; why would it happen?

**Review by bbengfort — APPROVED** [review]
Nice - looking forward to running this in staging!

File: cmd/bff/main.go (hunk 2 above)
Comment (bbengfort) [reply cmd/bff/main.go:346]: makes sense; thanks!

**Comment by bbengfort** [comment]
@pdeziel closing this as we've run the migration!

**Pull request closed without merging**